src/routes/docs/products/databases/queries/+page.markdoc

```diff
@@ -175,6 +175,219 @@ Query.select(["name", "title"])
 ```
 {% /multicode %}
 
+### Select relationship data {% #relationship-select %}
+
+With [opt-in relationship loading](/docs/products/databases/relationships#performance-loading), you must explicitly select relationship data. This gives you fine-grained control over performance and payload size.
+
+#### Get documents without relationships
+By default, documents return only their own fields:
+
+{% multicode %}
+```client-web
+const doc = await databases.getDocument('<DATABASE_ID>', '<COLLECTION_ID>', '<DOCUMENT_ID>');
+```
+```client-flutter
+final doc = await databases.getDocument(
+    databaseId: '<DATABASE_ID>',
+    collectionId: '<COLLECTION_ID>',
+    documentId: '<DOCUMENT_ID>'
+);
+```
+```python
+doc = databases.get_document('<DATABASE_ID>', '<COLLECTION_ID>', '<DOCUMENT_ID>')
+```
+```ruby
+doc = databases.get_document('<DATABASE_ID>', '<COLLECTION_ID>', '<DOCUMENT_ID>')
+```
+```deno
+// Returns only main document fields
+const doc = await databases.getDocument('<DATABASE_ID>', '<COLLECTION_ID>', '<DOCUMENT_ID>');
+// Result: { name: "John", age: 30 } - no related data
+```
+```php
+$doc = $databases->getDocument('<DATABASE_ID>', '<COLLECTION_ID>', '<DOCUMENT_ID>');
+```
+```swift
+let doc = try await databases.getDocument(
+    databaseId: "<DATABASE_ID>",
+    collectionId: "<COLLECTION_ID>",
+    documentId: "<DOCUMENT_ID>"
+)
+```
+```http
+# Returns only main document fields
+GET /v1/databases/<DATABASE_ID>/collections/<COLLECTION_ID>/documents/<DOCUMENT_ID>
+# Result: { name: "John", age: 30 } - no related data
+```
+{% /multicode %}
+
+#### Load all relationship data
+Use the `*` wildcard to load all fields from related documents:
+
+{% multicode %}
+```client-web
+const doc = await databases.getDocument(
+    '<DATABASE_ID>', '<COLLECTION_ID>', '<DOCUMENT_ID>',
+    [Query.select(["*", "reviews.*"])]
+);
+```
+```client-flutter
+final doc = await databases.getDocument(
+    databaseId: '<DATABASE_ID>',
+    collectionId: '<COLLECTION_ID>',
+    documentId: '<DOCUMENT_ID>',
+    queries: [Query.select(["*", "reviews.*"])]
+);
+```
+```python
+doc = databases.get_document(
+    '<DATABASE_ID>', '<COLLECTION_ID>', '<DOCUMENT_ID>',
+    [Query.select(["*", "reviews.*"])]
+)
+```
+```ruby
+doc = databases.get_document(
+    '<DATABASE_ID>', '<COLLECTION_ID>', '<DOCUMENT_ID>',
+    [Query.select(["*", "reviews.*"])]
+)
+```
+```deno
+const doc = await databases.getDocument(
+    '<DATABASE_ID>', '<COLLECTION_ID>', '<DOCUMENT_ID>',
+    [Query.select(["*", "reviews.*"])]
+);
+// Result: { name: "John", age: 30, reviews: [{...}] }
```

Comment:
only for deno?

Reply:
We've been using only Deno in this queries doc, but Node.js makes more sense for the examples. I've updated it